contrib/kind-common

```diff
@@ -168,13 +169,15 @@ EOF
   # Override GOBIN until https://github.com/metallb/metallb/issues/2218 is fixed.
   GOBIN="" inv dev-env -n ovn -b frr -p bgp -i "${ip_family}"
 
+  docker network rm clientnet || true
```

Comment:
`docker network rm -f` instead of the `|| true`

```
Options:
  -f, --force   Do not error if the network does not exist
```

Reply:
Just a super nit, good for follow ups, 
LGTM

Reply:
will follow up